README.md

```diff
@@ -1,59 +1,117 @@
 # Post-it-app
+
+[![Build Status](https://travis-ci.org/Charpell/post-it-app.svg?branch=Google_Signup_Refactor)](https://travis-ci.org/Charpell/post-it-app)
+
+[![Coverage Status](https://coveralls.io/repos/github/Charpell/post-it-app/badge.svg?branch=Google_Signup_Refactor)](https://coveralls.io/github/Charpell/post-it-app?branch=Badges)
+
+
+
+
 PostIt is a simple application that allows friends and colleagues create groups for notifications. This way one person can post notifications to everyone by sending a message once. The application allows people create accounts, create groups and add registered users to the groups, and then send messages out to these groups whenever they want.
 
+# Features
+  1. Sign Up
+  2. Log in
+  3. Create group
+  4. Add Member to group
+  5. Send broadcast message to group members
+
+
 
 ## Prerequisites
     Node Enviroment 
     Git 
     Firebase Database Account
     Postman
-    Google Chrome
+    Google Chrome 
 
 
-## Deployment
+# Deployment
  PostIt is hosted on heroku and can be accessed via this link
-    https://post-it-app35.herokuapp.com/
+        https://post-it-app35.herokuapp.com/
 
 
+# Installation
+  Kindly follow the steps below to setup a local development environment.
+  1. Clone this repository from a terminal git clone https://github.com/Charpell/post-it-app.git.
+  2. Move into the project directory
+  3. Install project dependencies npm install
+  4. Open 2 git bash terminal, one will run the server and the other runs webpack
```

Comment:
Noted and Implemented. However instead of using two terminal windows, i switched to one